lua/nvim-tree/git/runner.lua

```diff
@@ -100,11 +122,20 @@ function Runner.run(opts)
     list_ignored = opts.list_ignored,
     timeout = opts.timeout or 400,
     output = {},
-    _done = false
+    rc = nil, -- -1 indicates timeout
```

Comment:
Care has been taken to not alter the functionality or flow of control here. _done -> rc is the only significant change.